databases/week2/part3_queries.sql

```diff
@@ -0,0 +1,48 @@
+
+--Part 3: More queries
```

Comment:
the comments have to have space after -- otherwise most of the IDEs will not recognize it as a comment but syntax error :)